tests/python/hamiltonians/molecular_hamiltonian_test.py

```diff
@@ -162,3 +162,25 @@ def test_rotated():
     original_expectation = np.vdot(vec, linop @ vec)
     rotated_expectation = np.vdot(rotated_vec, linop_rotated @ rotated_vec)
     np.testing.assert_allclose(original_expectation, rotated_expectation)
+
+
+def test_from_fermion_operator():
+    norb = 5
```

Comment:
Make the the test parametrized (see other tests for examples) and test a range of `norb` (0 to 4 should be good enough).